compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/AbstractTypeSymbol.java

```diff
@@ -134,7 +134,7 @@ public BType getBType() {
         List<FunctionSymbol> filteredFunctions = new ArrayList<>();
 
         for (FunctionSymbol function : functions) {
-            ParameterSymbol firstParam = function.typeDescriptor().parameters().get(0);
+            ParameterSymbol firstParam = function.typeDescriptor().params().get().get(0);
```

Comment:
Can the `typeDescriptot().params()` return `empty`? According to `FunctionTypeSymbol` implementation, it can right?

Reply:
In this particular context, no. It only return empty when it represents the `function` typedesc. Since here, we are getting the function type symbol from a `FunctionSymbol` instance, it should never return empty.

Reply:
No issue then :+1: